Add alt text (#21941)
mdn/content: files/en-us/web/api/element/scrollheight/index.md

measurement of the height of an element's content, including content not visible on the
screen due to overflow.

> ![Scroll height example featuring an ad for cat adoption](scrollheight.png)

Reviewer: This image conveys a lot of information that the sighted user can access but is not available to the screen reader user. It shows that the scroll height goes from the container's padding top to the end of the padding bottom, well beyond the top and bottom of the viewport. That should be conveyed in the alt text. The cat part isn't important.

Author: @estelle I have added a more descriptive alt text.

> ![We see an example element with four regions labeled padding-top, border-top, border-bottom, padding-bottom. The scroll height goes from the container's padding top to the end of the padding bottom, well beyond the top and bottom of the viewport.](scrollheight.png)

Reviewer: ```suggestion
![The user's veiwport is an element with four regions labeled padding-top, border-top, border-bottom, padding-bottom. The scroll height goes from the container's padding top to the end of the padding bottom, well beyond the top and bottom of the viewport.](scrollheight.png)
```
